linchpin/__init__.py

```diff
@@ -707,6 +705,69 @@ def do_action(self, provision_data, action='up', run_id=None, tx_id=None):
         return (return_code, lp_data)
 
 
+    def get_run_data(self, tx_id, fields, targets=()):
+        """
+        Returns the RunDB for data from a specified field given either a run_id
+        or tx_id. The fields consist of the major sections in the RunDB (target
```

Comment:
Is this correct?  it says either a run_id or tx_id but I only see tx_id passed in to the method.

Reply:
tx_id holds the run_ids for a particular run (or transaction). To be able to get the data, the tx_id is used to get the run_ids. In turn, the resource data is pulled by the run_id into a returnable format.

Reply:
I see what you mean. I missed the point. The docs just need to remove the reference to the run_id. I'll get that fixed in the next update coming shortly.